scripts/logging.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+#
+# Helper functions for console (stdout/stderr) logging.
+#
+
+COLOR_GREEN="\033[0;32m"
+COLOR_YELLOW="\033[0;33m"
+COLOR_RED="\033[0;31m"
+COLOR_RESET="\033[0m"
+
+# Helper function, not to use directly.
+# Emits the current timestamp to stdout to be used as header in logging traces.
+# If `date` command fails, it will print the error output instead and will
+# exit with status code 1.
+timestamp() {
+    local tms
```

Comment:
```suggestion
	local tms
```

Reply:
Fixed in https://github.com/test-network-function/cnf-certification-test-partner/pull/283/commits/1116e03e19d4a3529db7c7322d3616bdd4a43c87